playbooks/common/openshift-checks/health.yml

```diff
@@ -11,4 +11,4 @@
     - action: openshift_health_check
       args:
         checks:
-          - '@preflight'
+          - '@health'
```

Comment:
I think that the idea here is for checks like #3660 (etcd image metadata size) to match that tag... i.e. checks that are meaningful for a running cluster, like certificate expiration...